engines/sci/detection.cpp

```diff
@@ -103,6 +103,8 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"gk1demo",         "Gabriel Knight: Sins of the Fathers"},
 	{"qfg4demo",        "Quest for Glory IV: Shadows of Darkness"},
 	{"pq4demo",         "Police Quest IV: Open Season"},
+	// === SCI1.6 and SCI1.6+ games ===========================================
+	{"catdate",         "The Dating Pool"},
```

Comment:
It's an interpreter that's basically for all intents and purposes SCI11 but it has SCI2's kRemapColors in it. It came in an archive named "SCI16" and it's certainly a little later than SCI11, so I went with that name in conversation and added a version dot when I wrote the highlighted line to indicate "this does _not_ refer to it being 16-bit". I'd wager it might even be a little later than the QfG4 demo, considering the details on kRemapColors.

I'm not the only one with access to this interpreter so more fanmade games that use it may follow, from other creators, but I'm not aware of any projects. Most seem to favor plain, vanilla SCI11 now that we have the tools.

I added a few _other_ things on top of that, which fall under the + bit, like the outline strokes in kDisplay. That version is also available for others to use in their fangames, should they wish to.

Reply:
It's still SCI1.1 as far as I am concerned. The changelog in the now-public repository ends at 1.001.097, but that is not the latest known SCI1.1 interpreter. LSL6 has 1.001.113 and Freddy Pharkas has 1.001.132, and we don't have those changes. It seems reasonable to assume that these changes were made in parallel, and the RemapColors call compiled out because it took up memory and was not used (I'm unsure how much heap exactly, but I know that Sierra went to great lengths to conserve heap space). QfG4demo uses it because it is (surprise!) a demo of an SCI32 game.

It is however interesting to note that the LSL6 palettes are laid out as though the RemapColors feature existed (i.e. the last 20 entries are encoded as bright red and in practice are reserved). Of course, Windows's handling of 8-bit palettes is another factor that makes this choice reasonable.

Reply:
What I'm saying is, SCI1.1 with fanmade changes is a better way to describe it.

Reply:
Y'know, that's a very good argument that I wish I'd seen way back then. You've certainly convinced me now.